GLOSSARY.md

```diff
@@ -0,0 +1,31 @@
+<dl>
+  <dt>GPR</dt>
+  <dd>gaussian process regression</dd>
+
+  <dt>SVGP</dt>
+  <dd>stochastic variational inference for Gaussian process models</dd>
+
+  <dt>[] shape constructions</dt>
+  <dd>the way of describing tensor shape in docstrings and comments. Example: [..., N, D, D], this is tensor with unknown number of leading dimensions and last two dimensions are equal</dd>
+
+  <dt>X</dt>
+  <dd>and its variations like Xnew in GPflow code mean input data (numpy or tensorflow)</dd>
```

Comment:
```suggestion
  <dd>(and variations like Xnew) refers to input points; always of rank 2, e.g. shape [N, D], even when D=1</dd>
```
(I think a lot of the entries can describe both numpy arrays or tensorflow tensors so I think it's confusing if this is the only place where it's mentioned explicitly)